Website/Packages/datax-pipeline/src/modules/flowDefinition/flowReducer.js

```diff
@@ -26,9 +27,14 @@ const INITIAL_FLOW_STATE = {
     outputTemplates: [],
     rules: [],
 
+    batchInputs: [Models.getDefaultBatchInputSettings()],
+    batchList: [],
+
     // State
     isNew: false,
     isDirty: false,
+    selectedFlowBatchInputIndex: undefined,
```

Comment:
For selectedFlowBatchInputIndex, besides seeing it set in FLOW_INIT, its actually not used anywhere else. I think you can remove this unneeded field as selectBatchIndex is the one you are actually using.

Reply:
I use selectedFlowBatchInputIndex.